arcgis-ios-sdk-samples/Maps/Create and save a map/README.md

```diff
@@ -27,6 +27,10 @@ Select the basemap and layers you'd like to add to your map. Tap the "Save" butt
 * AGSMap.save
 * AGSPortal
 
+## Additional information
+
+The sample allows users to view the basemaps with the pre-baked API keys. To save the map to ArcGIS Online, please login with an ArcGIS identity/named user account.
```

Comment:
```suggestion
The sample allows users to view the basemaps with the pre-baked API key. To save the map to ArcGIS Online, please login with an ArcGIS identity/named user account.
```

Reply:
I'd like to express the meaning of "app-wide key", "the key set on `ArcGISRuntimeEnvironment`", or "the default key" here. I don't think automated is the word that I'm looking for 😕

Reply:
I like "default key'!